README.md

```diff
@@ -267,6 +267,46 @@ dependencies:
   - someDep
 ```
 
+## Editor Integration
+
+Note to editor integrators: there is now a haskell-language-server-wrapper executable, which is installed alongside the haskell-language-server executable. When this is invoked in the project root directory, it attempts to work out the GHC version used in the project, and then launch the matching hie executable.
+
+All of the editor integrations assume that you have already installed HLS (see above) and that stack put the hls binary in your path (usually ~/.local/bin on linux and macOS).
+
+### Using HLS with Emacs
+
+Install HLS along with the following emacs packages:
+
+[lsp-mode](https://github.com/emacs-lsp/lsp-mode)
+[lsp-ui](https://github.com/emacs-lsp/lsp-ui)
+[lsp-haskell](https://github.com/emacs-lsp/lsp-haskell)
+
+Make sure to follow the instructions in the README of each of these packages.
+
+``` lisp
+(setq lsp-haskell-process-path-hie "haskell-language-server-wrapper")
```

Comment:
just for clarification, is this still correct? `lsp-haskell-process-path-hie`?

Reply:
i think so https://github.com/emacs-lsp/lsp-haskell/blob/master/lsp-haskell.el#L40